signer/src/stacks/api.rs

```diff
@@ -1296,6 +1386,298 @@ mod tests {
         first_mock.assert();
     }
 
+    #[tokio::test]
+    async fn get_pox_info_and_get_nakamoto_start_height_works() {
+        let raw_json_response = r#"
```

Comment:
nit: move to `tests/fixtures` and `include_str!` it?

Reply:
Good idea, [2bceba5](https://github.com/stacks-network/sbtc/pull/633/commits/2bceba5882b719400fc94fd0d9c1f950f812815c)